learn/what_is_meilisearch/telemetry.md

```diff
@@ -17,7 +17,8 @@ We use two data-collection tools: [Amplitude](https://amplitude.com/amplitude-an
 - Amount of time the MeiliSearch instance has been running
 - MeiliSearch version
 - Technical specs of the device running MeiliSearch
-- Number of documents per index
+- Database size per instance
+- Number of documents per instance
```

Comment:
According to @curquiza's PR #1065, `Number of documents per index` will only be available on v0.22.

Is `Number of documents per instance` available on v0.21? Or is the proposed addition only correcting exchanging `index` with `instance` but refer to the same field?

Reply:
Hey @guimachiavelli!

An instance can contain several indexes.

So the @curquiza fix is ok! In 0.21 we already have `Number of documents per instance`.

Reply:
Got it, thanks for clarifying!